examples/kms/vault/aws-credentials.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# This is an example Kubernetes Secret that can be created in the Kubernetes
+# Namespace where Ceph-CSI is deployed. The contents of this Secret will be
+# used to connect to the Amazon KMS.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-csi-aws-credentials
+stringData:
+  AWS_ACCESS_KEY_ID: "arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab"
+  AWS_SECRET_ACCESS_KEY: "JalrXUtnFEMI/K7MDENG/bPxRfiCYzEXAMPLEKEY"
+  AWS_SESSION_TOKEN: ""
```

Comment:
@nixpanic can we document about these and also mention which are mandatory/required?

Reply:
I have added this to `docs/deploy-rbd.md`.